.github/workflows/ci.yaml

```diff
@@ -92,28 +92,30 @@ jobs:
                 php-version:
                     - '8.0'
                 symfony-version:
-                    - '4.4.*'
-                    - '5.3.*'
+                    - '5.4.*'
                     - '5.4.x-dev'
                     - '6.0.x-dev'
                 dependency-versions: ['highest']
                 allow-dev-deps-in-apps: ['0']
                 include:
                     # testing lowest PHP version with LTS
```

Comment:
this comment doesn't match up: because this is testing highest dependencies.

Reply:
fixed